### PR TITLE
Feature/nt gk shell

### PR DESCRIPTION
**Pull request by kumy**
We should not forget to remove smarty and smarty getText from the base image Dockerfile https://github.com/geokrety/geokrety-website-docker-base/tree/smarty3

### CODE REVIEW

**Comment by boly38** [comment]
I just create geokrety/geokrety-website-docker-base#1 to not forget

**Comment by boly38** [comment]
I must remove some useless comment before merging, I'll do that at noon

File: website/templates/labels/label-2.php [hunk 1]
```diff
@@ -32,7 +32,6 @@
 $opis = mb_substr(strtr(strip_tags($opis), $conaco), 0, $cut);
 $id = sprintf('GK%04X', $id);
 
-define('SMARTY_DIR', '/usr/share/php/smarty/libs/');
```
Comment (boly38) [inline website/templates/labels/label-2.php]: I were not able to find where this file are used : always used ?

**Comment by kumy** [comment]
Manually merged in `new-theme`

**Pull request closed without merging**